Exiled.Events/Extra/Events/EventAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="EventAttribute.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Extra.Events
+{
+    using System;
+
+    /// <summary>
+    /// An attribute to mark listeners.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, Inherited = false)]
+#pragma warning disable SA1649
+    public sealed class ListenerAttribute : Attribute
```

Comment:
instead of using pragma here, why don't you rename the file to the class name, so you don't need pragmas here